recipes/custodian/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "custodian" %}
+{% set version = "2019.2.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18864727dcc279e5634d300940e3440df17a5151e426b9451f923c3246416973
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - cstdn = custodian.cli.cstdn:main
+    - run_vasp = custodian.cli.run_vasp:main
+    - run_nwchem = custodian.cli.run_nwchem:main
+    - converge_kpoints = custodian.cli.converge_kpoints:main
+    - converge_geometry = custodian.cli.converge_geometry:main
+
+requirements:
+  build:
```

Comment:
`build` -> `host`

Reply:
@isuruf But this breaks the build process - so I would revert your commit: 
`RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/numpy/`